<?xml version="1.0" ?>
<sdf version="1.5">
  <model name="blue_ball">
    <static>true</static>
    <link name="link">
      <inertial>
        <mass>0.5</mass>
        <inertia>
          <ixx>0.01</ixx>
          <ixy>0.0</ixy>
          <ixz>0.0</ixz>
          <iyy>0.01</iyy>
          <iyz>0.0</iyz>
          <izz>0.01</izz>
        </inertia>
      </inertial>
      
      <collision name="collision">
        <geometry>
          <sphere>
            <radius>0.15</radius>
          </sphere>
        </geometry>
      </collision>
      
      <visual name="visual">
        <geometry>
          <sphere>
            <radius>0.15</radius>
          </sphere>
        </geometry>
        <material>
          <ambient>0.0 0.0 1.0 1</ambient>
          <diffuse>0.0 0.4 1.0 1</diffuse>
          <specular>0.5 0.5 1.0 1</specular>
          <emissive>0 0 0.2 1</emissive>
        </material>
      </visual>
    </link>
  </model>
</sdf>